test/unit/lib/getPharmacies.js

```diff
@@ -74,7 +74,24 @@ describe('Nearby', () => {
     });
 
     it('should return the opening times message and open state', () => {
-      const results = pharmacies.nearby(searchPoint, geo).openServices;
+      const alwaysOpenOrg = {
```

Comment:
I'm guessing we don't have a closed test due to the fact we can't currently write a reliable test which will not fail depending on the time the test is run. I've added an issue for this (#154)